google-cloud-spanner/src/test/java/com/google/cloud/spanner/RandomResultSetGenerator.java

```diff
@@ -142,6 +147,16 @@ private void setRandomValue(Value.Builder builder, Type type) {
                   random.nextInt(2019) + 1, random.nextInt(11) + 1, random.nextInt(28) + 1);
           builder.setStringValue(date.toString());
           break;
+        case INTERVAL:
+          Interval interval =
+              Interval.builder()
+                  .setMonths(10)
+                  .setDays(45)
+                  .setMicros(-89)
+                  .setNanoFractions((short) 45)
+                  .build();
+          builder.setStringValue(interval.ToISO8601());
+          break;
```

Comment:
This is supposed to generate a random value that varies per test run.

Reply:
changed.